charts/shenyu/README.md

```diff
@@ -6,14 +6,151 @@ Helm deployment documentation written for Apache/Shenyu.
+
+### 2. How to install only admin or bootstrap
+
+* Install only admin: add `-set bootstrap.enabled=false` to the end of the helm install command
+* Install only bootstrap: add `--set admin.enabled=false` to the end of the helm install command
+
+### 3. How to install old version ShenYu
+
+```shell
+helm search repo shenyu -l
+```
+
+You will get output similar to
+
+```shell
+NAME CHART VERSION APP VERSION DESCRIPTION
+shenyu/shenyu 0.2.0 2.5.0 Helm Chart for deploying Apache ShenYu in Kubernetes
+...
+...
+```
+
+where `APP_VERSION` is the version of ShenYu and `CHART_VERSION` is the version of Helm Chart.
+
+Select the corresponding Chart version according to the version of ShenYu you want to install, and add the `-version=CHART_VERSION` parameter at the end of the command. For example
+
+```shell
+helm install shenyu shenyu/shenyu -n=shenyu --version=0.2.0 --create-namespace
+```
+
+## Values configuration instructions
+
+### Global Configuration
+| configuration-item | type   | default                     | description                                                                                        |
+|--------------------|--------|-----------------------------|----------------------------------------------------------------------------------------------------|
+| replicas           | int    | `1`                         | Number of replicas                                                                                 |
+| version            | string | `"2.5.0"`                   | shenyu version, it is not recommended to modify, please install the corresponding version directly |
+| admin.enabled      | bool   | `true`                      | Whether to install shenyu-admin                                                                    |
+| admin.image        | string | `"apache/shenyu-admin"`     | shenyu-admin image                                                                                 |
+| admin.nodePort     | int    | `31095`                     | shenyu-admin NodePort                                                                              |
+| bootstrap.enabled  | bool   | `true`                      | whether to install shenyu-bootstrap                                                                |
+| bootstrap.image    | string | `"apache/shenyu-bootstrap"` | shenyu-bootstrap image                                                                             |
+| bootstrap.nodePort | int    | `31195`                     | shenyu-bootstrap NodePort                                                                          |
+
+### Database configuration
+
+#### General database configuration
+| configuration-item     | type   | default | description                                     |
+|------------------------|--------|---------|-------------------------------------------------|
+| dataSource.active      | string | `"h2"`  | Database to use, supports `h2`, `mysql`         |
```

Comment:
Database to use, supports `h2`, `mysql` 

add `PostgreSQL `

Reply:
Got it